plugin/index.tpl.html

```diff
@@ -28,7 +28,6 @@ <h1>
 			<a href="index.html?tag=[tag]" property="tag" mv-attribute="null" mv-multiple></a>
 		</div>
 
-		<meta property="repo" />
```

Comment:
I was not suggesting we delete this. I was suggesting we make its value visible, but then again, we have a "repo" button anyway.

Reply:
Oops. If I understand you correctly, I should return both the `repo` and the `url` fields. But do we really need to make them visible? You fairly mentioned that we have a "repo" button anyway.